nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-0-11-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -45,6 +45,20 @@ The following binary components are provided under the Apache Software License v
       Apache Kafka
       Copyright 2012 The Apache Software Foundation.
 
+  (ASLv2) Yammer Metrics
```

Comment:
I'm assuming this is required by Kafka?

Reply:
I'll remove Yammer Metrics from the NOTICE files for the 0-9, 0-10, 0-11, 1-0, and 2-0 NARs.  After a second look, I see that the `kafka_2.x` dependencies are `test`-scoped.